### PR TITLE
Controller

### PR DESCRIPTION
**Pull request by parks38**
added controller file

### CODE REVIEW

File: WebMvc/Controllers/CartController.cs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            try
+            {
+                if (productDetails.Id != null)
+                {
+                    var user = _identityService.Get(HttpContext.User);
+                    var product = new CartItem()
+                    {
+                        Id = Guid.NewGuid().ToString(),
+                        Quantity = 1,
+                        ProductName = productDetails.Name,
```
Comment (rfeseha) [inline WebMvc/Controllers/CartController.cs]: EventName

File: WebMvc/Controllers/CartController.cs [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                        PictureUrl = productDetails.PictureUrl,
+                        UnitPrice = productDetails.Price,
+                        ProductId = productDetails.Id
```
Comment (rfeseha) [inline WebMvc/Controllers/CartController.cs]: EventId

**Review by rfeseha — COMMENTED** [review]
I changed couple of things in there ProductId to EventId and ProductName to EventName

**Review by vlau26 — APPROVED** [review]

**Comment by vlau26** [comment]
I merged it since Rebka and I already reviewed, need to move forward with the controller

**Pull request merged**